packages/eui/src/components/steps/step.styles.ts

```diff
@@ -66,6 +67,11 @@ export const euiStepStyles = (euiThemeContext: UseEuiTheme) => {
         background-position: -${euiTheme.size.xs} ${euiTheme.size.l};
       }
     `,
+    xxs: css`
+      &:not(:last-of-type) {
+        background-position: -${euiTheme.size.s} ${euiTheme.size.base};
```

Comment:
Not totally sure, but should this be `numberXXSSize` instead of `euiTheme.size.base`? The vertical offset is based on the indicator size, right?

```suggestion
        background-position: -${euiTheme.size.s} ${euiStep.numberXXSSize};
```

Reply:
The above is definitely not a blocking comment, it looks like the other sizes in this block should likely be using the step variables as well but aren't haha. Up to you if you'd like to change them all to be a bit more DRY or leave them as-is

Reply:
AH yeah, that can be improved. It's the same value but it makes it clearer if we use the step size 👍

Reply:
I updated to use the specific step size variables instead in this [commit](https://github.com/elastic/eui/pull/7813/commits/b55740dac21b617215f3e84b6957105b84080d60), thanks for bringing it up!

Reply:
Beautiful!!! 🌈 ✨